website/static/js/wikiSettingsTreebeard.js

```diff
@@ -0,0 +1,221 @@
+'use strict';
+
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+var Fangorn = require('js/fangorn');
+require('../css/fangorn.css');
+
+// TODO: Move 'resolveToggle' to a central location
+function resolveToggle(item) {
```

Comment:
`resolveToggle` should be moved to a central location such as `fangorn.js`, but might be out of the scope of this PR because this touches many other JS files. Should I?